emscripten.py

```diff
@@ -2763,6 +2763,10 @@ def load_metadata_wasm(metadata_raw, DEBUG):
   unexpected_exports = [e for e in unexpected_exports if e not in shared.Settings.EXPORTED_FUNCTIONS]
   shared.Building.user_requested_exports += unexpected_exports
 
+  # With the wasm backend the set of implemented functions is identical to the set of exports
+  # Set this key here simply so that the shared code that handle it.
+  metadata['implementedFunctions'] = [asmjs_mangle(x) for x in metadata['exports']]
```

Comment:
Is `metadata['implementedFunctions']` expected to be mangled but not `metadata['exports']`?

Reply:
Yeah, if you look at what binaries was producing you'll see it outputs the some set of value but one mangled and one not.